backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/tickets.test.js

```diff
@@ -97,6 +104,8 @@ const testValidateNewTicket = () => {
 				await next();
 			});
 
+			TemplatesSchema.generateFullSchema.mockImplementationOnce((template) => template);
```

Comment:
expect(generateFullSchema).toHaveBeenCalledTimes is missing in some tests in this file

Reply:
I changed my mind, moved the `generateFullSchema` call inside processor

Worried about regression if we have future functions calling them in different places they may not generate the full schema before calling.